examples/micro_speech/main/audio_provider.cc

```diff
@@ -110,7 +111,6 @@ static void CaptureSamples(void* arg) {
   return;
 #else
   size_t bytes_read = i2s_bytes_to_read;
-  uint8_t i2s_read_buffer[i2s_bytes_to_read] = {};
```

Comment:
The original code place this big buffer locally and therefore stored in stack. This potentially cause stack overflow like in my case. Please note in real application, we have multiple tasks needed to define apart from the task in this example.